src/containers/AdminPhoneNumberInventory.js

```diff
@@ -138,6 +171,18 @@ class AdminPhoneNumberInventory extends React.Component {
         label: "Available",
         style: inlineStyles.column
       },
+      {
+        key: "deleteButton",
+        label: "",
+        style: inlineStyles.column,
+        render: (columnKey, row) =>
+          this.props.params.ownerPerms ? (
+            <FlatButton
```

Comment:
Why not an `IconButton`?

Reply:
I initially tried `IconButton` but it didn't look right to me. No hover state and the tool tip was getting cut off by the row, which I'm sure is fixable but `FlatButton` seemed fine.